packages/shared/styles/components/SfAlert.scss

```diff
@@ -5,16 +5,19 @@ $sf-alert-font-size                 : 0.875rem !default;
 
 $sf-alert__text-margin              : 0 !default;
 
-$sf-alert__icon-padding-right       : 0.625rem !default;
+$sf-alert__text-padding-left       : 0.625rem !default;
 
-$sf-alert--info-background-color    : $c-yellow-secondary !default;
-$sf-alert--info-color               : $c-yellow-primary !default;
+$sf-alert--info-background-color    : #fdf1d7 !default;
+$sf-alert--info-color               : $c-black !default;
```

Comment:
```suggestion
$sf-alert--info-color               : $c-dark-primary !default;
```
@mayashavin in designe we use $c-dark-primary ;)
![color](https://user-images.githubusercontent.com/12138170/60509622-aca16d80-9ccd-11e9-823b-8f1d2e2b6803.png)

Reply:
to make it more automated and easy to customizable without losing accessibility, i think we should use a YIQ function here, @mayashavin @przspa  wdyt?
something like:
```scss
$sf-alert--info-color: color-yiq($sf-alert--info-background-color, $c-dark-primary, $c-white-primary) !default;
```

Reply:
I think we should :)

Reply:
I'm purposing common color modifiers using YIQ by default [here](https://github.com/DivanteLtd/storefront-ui/pull/221), so may be a good idea to merge my PR first, then use these new classes, wdyt?

Reply:
Although it may be necessary to replace some styles for `SfAlert` color modifiers, default helpers are perfect for button but not for alert I think (looking for the designs as reference)

Reply:
@leomp12 I think since we changed the design, there is no need for `color-yiq` for now. We don't have to wait for your PR :)